cockatrice/src/main.cpp

```diff
@@ -97,6 +100,19 @@ bool settingsValid()
         !settingsCache->getPicsPath().isEmpty();
 }
 
+void generateClientID()
```

Comment:
Could you give me an example of what would need refactored to work around it?  Returning the value then immediately storing it makes almost no sense to me as all it does is require additional code line.  Its not that big a deal one way or the other IMO and I can see were you may want consistency in the code since the other functions through out the code do as youyou describe.